nixos/modules/services/databases/redis.nix

```diff
@@ -88,6 +88,13 @@ in
         example = "/run/redis/redis.sock";
       };
 
+      unixSocketPerm = mkOption {
```

Comment:
Personally I would just recommend users to use `services.redis.settings.unixsocketperm = 750;` but this is fine :+1: It matches the `unixSocket` option nicely.